packages/prover/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM ubuntu:22.04
```

Comment:
@shreyas-londhe Could you update the Dockerfile to be compatible with the gpu prover?
Reference: https://github.com/zkemail/ether-email-auth/blob/main/packages/prover/Dockerfile#L1-L85